source/releasenotes/2025-05-22-pantheon-apachesolr-1-1-1.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Local bug fix update for Pantheon Apache Solr (7.x-1.1.1) in the Drupal 7 upstream
+published_date: "2025-05-22"
+categories: [drupal]
+---
+
+A followup to [last week's update](/release-notes/2025/05/drops-7-solr-update) for Drupal 7's `pantheon_apachesolr` module has been released to the Drupal 7 upstream.
+
+This update resolves a regression with local development in version 1.1 of `pantheon_apachesolr`.
+
+As a reminder, customers using Solr with Drupal 7 sites should update their sites to the latest version of the upstream as soon as possible. Customers with custom upstreams will need to update their upstreams to [include the last changes in the platform upstream](https://github.com/pantheon-systems/drops-7/compare/7.103.3...7.103.5).
```

Comment:
Do we want or expect people to read this diff? https://github.com/pantheon-systems/drops-7/compare/7.103.3...7.103.5